### PR TITLE
Media - Remove upload functionality from Add photos function

### PR DESCRIPTION
**Pull request by liviu-preda-tsystems**
https://jira.telekom.de/browse/NMC-2761
image
image

### CODE REVIEW

File: css/apps/media.scss [hunk 1]
```diff
@@ -273,4 +273,33 @@
 			}
 		}
 	}
+
+	.file-picker__actions {
```
Comment (aleksei-nmc) [inline css/apps/media.scss:277]: please write a comment to which modal window this styles are applied

File: css/apps/media.scss [hunk 2]
```diff
@@ -273,4 +273,33 @@
...
+		flex-direction: row-reverse;
+	
+		.upload-picker {
+			display: none;
+		}
+	
+		button {
+			&:disabled {
+				&>.button-vue__wrapper>.button-vue__icon>.material-design-icon.image-plus-icon {
```
Comment (aleksei-nmc) [inline css/apps/media.scss:286]: no need for such long selectors, just .material-design-icon.image-plus-icon will be enough

File: css/apps/media.scss [hunk 3]
```diff
@@ -273,4 +273,33 @@
...
+					background-image: var(--icon-add-dark);
+				}
+			}
+	
+			&>.button-vue__wrapper>.button-vue__icon>.material-design-icon.image-plus-icon {
```
Comment (aleksei-nmc) [inline css/apps/media.scss:291]: same here for selector length

File: css/apps/media.scss [hunk 4]
```diff
@@ -273,4 +273,33 @@
...
+				background-image: var(--icon-add-white);
+				width: 44px;
+				height: 44px;
+				background-position: center;
+				background-repeat: no-repeat;
+				background-size: 24px;
+			}
+		}
+	
+		svg {
```
Comment (aleksei-nmc) [inline css/apps/media.scss:301]: we want to affect the icon of only one button, this selector though will hide any svg that appear in .file-picker__actions

**Review by aleksei-nmc — CHANGES_REQUESTED** [review]

File: css/apps/media.scss (hunk 1 above)
Comment (liviu-preda-tsystems) [reply css/apps/media.scss:277]: Done

File: css/apps/media.scss (hunk 2 above)
Comment (liviu-preda-tsystems) [reply css/apps/media.scss]: Fixed

File: css/apps/media.scss (hunk 3 above)
Comment (liviu-preda-tsystems) [reply css/apps/media.scss]: Fixed

File: css/apps/media.scss (hunk 4 above)
Comment (liviu-preda-tsystems) [reply css/apps/media.scss]: Fixed

**Review by aleksei-nmc — APPROVED** [review]

**Pull request merged**